lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -62,6 +62,13 @@ class GoogleCloudOutput < BufferedOutput
     config_param :vm_id, :string, :default => nil
     config_param :vm_name, :string, :default => nil
 
+    # Associate logs with the compute service of the platform rather than
+    # attempting to detect if the VM is owned by a subservice such as App Engine
+    # or Kubernetes. This currently only has any effect when running on GCE.
+    # The initial motivation for this is to separate out Kubernetes node
+    # component (Docker, Kubelet, etc.) logs from container logs.
+    config_param :detect_subservice, :bool, :default => false
```

Comment:
Haha well technically combined with still using `unless` below the default was working as intended :)